sklearn/feature_selection/_univariate_selection.py

```diff
@@ -436,6 +436,18 @@ def f_regression(X, y, *, center=True, force_finite=True):
     SelectFwe: Select features based on family-wise error rate.
     SelectPercentile: Select features based on percentile of the highest
         scores.
+        
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from sklearn.feature_selection import f_regression
+    >>> X = np.array([[1, 2 ],[3,  4], [5, 6], [7,  8]])
+    >>> y = np.array([  7,  99,  14,  88])
+    >>> f_values, p_values = f_regression(X, y)
+    >>> f_values
+    array([0.43599148, 0.43599148])
+    >>> p_values
+    array([0.57694083, 0.57694083])
```

Comment:
Let's generate from the `make_regression` function. We use also the ellipsis (`...`) to avoid small fluctuation that could happen depending on the platform.

```suggestion
    >>> from sklearn.datasets import make_regression
    >>> from sklearn.feature_selection import f_regression
    >>> X, y = make_regression(
    ...     n_samples=50, n_features=3, n_informative=1, noise=1e-4, random_state=42
    ... )
    >>> f_statistic, p_values = f_regression(X, y)
    >>> f_statistic
    array([1.2...+00, 2.6...+13, 2.6...+00])
    >>> p_values
    array([2.7..., 1.5..., 1.0...])
```